.github/workflows/build.yml

```diff
@@ -98,3 +98,38 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  test:
+    name: Run Go Package Tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Download Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '^1.20.2'
```

Comment:
How does everyone feel about using a matrix of the current `stable` and `oldstable` versions of Go?  Right now, I believe those are `1.20.3` and `1.19.8`, per https://go.dev/dl/.

If so, the test name above should include the Go version number like we do in the Python tests.

Reply:
Will wait for additional input.

Reply:
Adding on to this I would prefer we just use the `stable` and `oldstable` minor releases (so `1.20` and `1.19`) rather than obfuscating the logic using a caret range. The action should pull the latest patch release for the provided minor version by default. This mirrors how we [configure `cisagov/skeleton-python-library`](https://github.com/cisagov/skeleton-python-library/blob/65efbf63dbfc4f2dfaec9190d1ec9448af565f64/.github/workflows/build.yml#L113-L118) to [install Python](https://github.com/cisagov/skeleton-python-library/blob/65efbf63dbfc4f2dfaec9190d1ec9448af565f64/.github/workflows/build.yml#L124-L127).

Reply:
What @mcdonnnj said makes sense to me as well.

Reply:
This looks good now. 👍